test/integration/const.go

```diff
@@ -4,6 +4,6 @@ import "time"
 
 const (
 	ciDelayCoefficient = 1
-	defaultTimeout     = 60 * time.Second * ciDelayCoefficient
+	defaultTimeout     = 2 * time.Minute * ciDelayCoefficient
```

Comment:
Oh, bumping up timeouts. Not great.

Reply:
Tests on Azure are a little bit slower than on packet. So I were seeing timeout-failures from time to time. Probably we'll need to scale AKS nodes in future. Currently we use [Standard_B2s](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#b-series).